views/login.hbs

```diff
@@ -0,0 +1,95 @@
+
+<h4>Login</h4>
+<script>  
+  ;(function(ns){
+    
+     //private
+    
+    function resetErrors(){
```

Comment:
You might be able to get the form instance itself and call `form.reset()` here, which is a native HTML Form Element function